micromixin-transformer/src/main/java/org/stianloader/micromixin/transform/internal/annotation/MixinShadowAnnotation.java

```diff
@@ -18,23 +20,24 @@
 import org.stianloader.micromixin.transform.internal.util.Objects;
 
 public final class MixinShadowAnnotation<T extends ClassMemberStub> extends MixinAnnotation<T> {
-
     @NotNull
     private final String prefix;
     @NotNull
     private final List<String> aliases;
+    private final boolean isMutable;
 
-    private MixinShadowAnnotation(@NotNull String prefix, @NotNull List<String> aliases) {
+    private MixinShadowAnnotation(@NotNull String prefix, @NotNull List<String> aliases, boolean isMutable) {
         this.prefix = prefix;
         this.aliases = aliases;
+        this.isMutable = isMutable;
     }
 
     @NotNull
-    public static <T0 extends ClassMemberStub> MixinShadowAnnotation<T0> parse(@NotNull AnnotationNode annotation) {
+    public static <T0 extends ClassMemberStub> MixinShadowAnnotation<T0> parse(@NotNull AnnotationNode annotation, @Nullable List<AnnotationNode> allAnnotations) {
         String prefix = "shadow$";
         List<String> aliases = Collections.emptyList();
         if (annotation.values == null) {
-            return new MixinShadowAnnotation<T0>(prefix, aliases);
+            return new MixinShadowAnnotation<T0>(prefix, aliases, false);
```

Comment:
If the `@Shadow` has no prefix or alias, then `@Mutable` will get discarded. This is probably not what you want to achieve.